apps/server/src/shared/domain/types/input-format.types.ts

```diff
@@ -1,8 +1,6 @@
 export enum InputFormat {
 	PLAIN_TEXT = 'plainText',
-	RICH_TEXT = 'richText',
-	RICH_TEXT_SIMPLE = 'inline',
+	RICH_TEXT_NEWS = 'richTextNews',
 	RICH_TEXT_CK4 = 'richTextCk4',
 	RICH_TEXT_CK5 = 'richTextCk5',
```

Comment:
I agree we have only 3 formats in the end, but "NEWS" is not a good name. Use the old name RICH_TEXT_CK5_SIMPLE I think makes more sense